cmd/controllers.go

```diff
@@ -71,5 +93,11 @@ func newControllersCmd(stopCh <-chan struct{}, client k8sclient.Interface) *cobr
 			return nil
 		},
 	}
+	addPortFlag(cmd, &port)
 	return cmd
 }
+
+// addTimeoutFlag adds timeout flag to command
```

Comment:
This comment doesn't seem to match the function

Reply:
fixed